### PR TITLE
feat: start creating CI for extract translation

### PR DESCRIPTION
**Pull request by ga-devfront**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | It is currently difficult to update the translations. To do this, you must go to the BO of a shop containing the module in order to export the translations and then retrieve the file to place in our module. This PR is used to introduce an automatic export of translations from the CI.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Sponsor company   | @PrestaShopCorp
| How to test?      | Just push something on dev branch with new translations inside files (like twig or php). You can see a new PR apear to add this translations.

### CODE REVIEW

File: .github/workflows/extract-translations.yml
Comment (Quetzacoalt91) [inline .github/workflows/extract-translations.yml:16]: @jolelievre need your insights here. Is that okay as-is or do we have to move this repository?

File: .github/workflows/extract-translations.yml
Comment (Quetzacoalt91) [inline .github/workflows/extract-translations.yml:23]: Could be interesting to rely on event contents to make this workflow generic, instead of having to specify the name and organization everytime we copy this file in a new repository.

Comment (ga-devfront) [reply .github/workflows/extract-translations.yml]: Yes good idea ! It's done ! Thank you for you review ;)

File: .github/workflows/extract-translations.yml
Comment (jolelievre) [reply .github/workflows/extract-translations.yml:18]: I didn't even know about this repo 😅 We just need to check if there are no sensitive data in the repo (like old keys/secrets) And then I guess it could be moved to the open source organisation

But if it's not blocking your CI right now I think you can start by using this address and update it when the repo will be moved

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
This PR is adding a workflow, so no need to ask the QA to review anything here.

The failure in CI is known to be unrelated with the changes here. We have an issue with the upgrade of ps_facetedsearch that breaks the UI tests. This PR can be merged in the meantime to enable the whole process regarding management of translation files.

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
